src/md/compiler/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(MDCOMPILER_SOURCES
   import.cpp
   importhelper.cpp
   mdutil.cpp
+  newmerger.cpp
```

Comment:
Its use is pretty pervasive in regmeta.cpp, regmeta_compilersupport.cpp, and regmeta_emit.cpp, all under `#ifdef FEATURE_METADATA_EMIT_ALL`.

Should I go back, and just have ilasm ignore errors returned from m_pEmitter->DefinePermissionSet()? Seems like we wouldn't get a true round trip in that case.